pkg/network/node/runtime.go

```diff
@@ -31,6 +31,14 @@ func (node *OsdnNode) getRuntimeService() (kubeletapi.RuntimeService, error) {
 				// Wait longer
 				return false, nil
 			}
+
+			// Ensure the runtime is actually alive; gRPC may create the client but
+			// it may not be responding to requests yet
+			if _, err := runtimeService.ListPodSandbox(&kruntimeapi.PodSandboxFilter{}); err != nil {
```

Comment:
Kinda lame that we fetch the complete list of running pods and then throw it away, but it looks like there's no better method we could call.

Reply:
@smarterclayton yeah, a "Ping" or "are you alive" method call would be great.